README.md

```diff
@@ -0,0 +1,106 @@
+# Django Url-Shortener App
+
+## Setting up your Development Environment ##
+1. First, fork this repository to your account.
+
+2. Create a virtual environment on your machine. 
+    ```
+    virtualenv -p python3 env
+    ```
+    We recommend using python3-virtualenv. Any other packages would do fine though.
+
+3. Activate the newly created virtual environment:
+    ```
+    cd env
+    source bin/activate
+    ```
+
+4. Clone this repository (this would make rebasing easier).
+    ```
+    git clone https://github.com/modihere/url-shortener.git
+    ```
+    
+5. Install the dependencies for the project.
+    ```
+    cd website
+    pip3 install -r requirements.txt
+    ```
+    
+6. Run the migrations and collect staticfiles
+    ```
+    python3 manage.py migrate
+    python3 manage.py collectstatic
+    ```
+    
+7. Run the live development server on your machine and test it.
+    ```
+    python3 manage.py runserver
+    ```
+    Once the server is started, open http://127.0.0.1:8000 in a web browser.
+    Everything went well if the webpage loads correctly and you don't see any errors.
+    
+8. Add a remote to your forked repository. This remote will be needed to push your changes to your repo.
+    ```
+    git remote add myfork https://github.com/<username>/website.git
+    ```
+    
+9. Find an issue in this repository that you would like to and can fix.
+   Start working on an issue. Steps 10 and beyond will guide you in doing this.
+
+   **In case multiple people are working on the same issue, the Pull Request that is completed first shall be given**
+   **the highest preference. The reviews will be on a first come first service basis.**   
```

Comment:
You can omit these two lines because the owner will decide which PRs will be merged.